app/javascript/mastodon/features/audio/index.js

```diff
@@ -313,19 +309,14 @@ class Audio extends React.PureComponent {
   }
 
   handleLoadedData = () => {
-    const { autoPlay, currentTime, volume, muted } = this.props;
+    const { autoPlay, currentTime } = this.props;
 
     if (currentTime) {
       this.audio.currentTime = currentTime;
     }
 
-    if (volume !== undefined) {
-      this.audio.volume = volume;
-    }
-
-    if (muted !== undefined) {
-      this.audio.muted = muted;
-    }
+    this.audio.volume = 1;
+    this.audio.muted = false;
```

Comment:
I'm not sure if I should keep this part or if this is the right place.

Reply:
That's a good question. I think it should rather be in `setAudioRef`.